zeroex/orderwatch/decoder/event_decoder_test.go

```diff
@@ -96,6 +100,28 @@ func TestDecodeERC20Approval(t *testing.T) {
 
 }
 
+func TestDecodeERC20ApprovalForNonConformingImplementation(t *testing.T) {
+	var approvalLog types.Log
+	err := unmarshalLogStr(erc20ApprovalLogTest, &approvalLog)
+	if err != nil {
```

Comment:
What's the difference between this block and `require.NoError(t, err)`?

Reply:
the latter takes just 1 line, but I just copied and tweaked over the `TestDecodeERC20Approval` one, which uses the standard `err != nil` .

I guess if this were to change we could change all the previous occurences in all tests.